Mention sidecar injection labels in Kyma docs (#15435)
kyma-project/kyma: docs/04-operation-guides/operations/smsh-01-istio-enable-sidecar-injection.md

   kubectl label namespace {YOUR_NAMESPACE} istio-injection=enabled
   ```

> * To enable sidecar proxy injection for a Deployment, you can add this as either label or annotation to the Deployment configuration file: `sidecar.istio.io/inject: "true"`

Reviewer: ```suggestion
* To enable sidecar proxy injection for a Deployment, add this to the Deployment configuration file as either a label or an annotation: `sidecar.istio.io/inject: "true"`
```
Unless there is an alternative way to do it, remove the `you can` part.

> Please note that the Namespace label will take precedence before the Pod label or annotations.

Reviewer: ```suggestion
Note that the Namespace label takes precedence over the Pod label or annotation.
```
- Our [guidelines](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/02-style-and-terminology.md#voice-and-tone) say to avoid unnecessary words, such as `please`, that don't add anything. 
- Something takes precedence **over** something else. See the [Cambridge Dictionary](https://dictionary.cambridge.org/dictionary/english/precedence) for reference. 
- Did you put `annotations` in plural here on purpose?
